src/experiments/data/resolvers.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import { setExperiments } from './actions';
+import { EXPERIMENT_NAME_PREFIX } from './constants';
+
+export function* getExperiments() {
+	const storageItems = Object.entries({ ...window.localStorage }).filter(
+		(item) => {
+			if (item[0].indexOf(EXPERIMENT_NAME_PREFIX) === 0) {
```

Comment:
Looks like we can simply return the `item[0].indexOf(EXPERIMENT_NAME_PREFIX) === 0` without "if".